src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryPipelineSpark.java

```diff
@@ -169,9 +173,11 @@ private void experimentalInterpretation(final JavaSparkContext ctx,
                                             final Broadcast<SAMSequenceDictionary> broadcastSequenceDictionary,
                                             final String sampleId,
                                             final FindBreakpointEvidenceSpark.AssembledEvidenceResults assembledEvidenceResults) {
-        final String expOutputDir =
-                IOUtils.getPath(vcfOutputFileName).getParent().toAbsolutePath().toString() +
-                        "/experimentalVariantInterpretations";
+        String expOutputDir =
+                IOUtils.getPath(vcfOutputFileName).getParent().toAbsolutePath().resolve(EXPERIMENTAL_INTERPRETATION_OUTPUT_DIR_NAME)
+                .toUri().toString();
+        if ( !BucketUtils.isRemoteStorageUrl(expOutputDir) )
+            expOutputDir = expOutputDir.replaceFirst("file://", "");
```

Comment:
At the very least I'd like to get rid of having to do this, it seems terribly hacky. You could at least query the URI above for the path minus the scheme, right?

Reply:
totally. Rewrote how the path is resolved in the new commit.